microshift_networking/microshift-network-policy/microshift-editing-network-policy.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-editing-network-policy"]
+= Editing a network policy
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-editing-network-policy
+
+toc::[]
+
+You can edit an existing network policy for a namespace.
+
+include::modules/nw-networkpolicy-edit.adoc[leveloffset=+1]
+
+include::modules/nw-networkpolicy-object.adoc[leveloffset=+1]
+
```

Comment:
I see this message when editing a network policy object directly

`[redhat@dhcp-1-235-172 ~]$ oc edit networkpolicy -n test2
A copy of your changes has been stored to "/tmp/oc-edit-2180362568.yaml"
error: At least one of apiVersion, kind and name was changed`
@ShaunaDiaz Is this expected behavior or pilot error ?

Reply:
Which fields were you changing? You can not change any of those 3 because they define the resource itself.

Reply:
@pacevedom I changed the name, so pilot error. Should this be noted in the document, or is this something a user should understand by using this command ?

When I modified just the port in my example, it worked as expected.

Reply:
Added a little explanatory text at the beginning of the assembly. Advanced users can just ignore it, but newer users might find it helpful.